src/azure-cli/azure/cli/command_modules/iot/custom.py

```diff
@@ -1416,6 +1416,126 @@ def _get_iot_central_app_by_name(client, app_name):
     return target_app
 
 
+def iot_central_list_private_link_resource(client, app_name=None, connection_id=None, resource_group_name=None, source_type=None):
+    if app_name and resource_group_name and source_type:
+        if source_type.lower() != 'microsoft.iotcentral/iotapps':
+            raise InvalidArgumentValueError(
+                "Type must be Microsoft.IoTCentral/iotApps")
+    elif connection_id:
+        id_list = connection_id.split('/')
+        if id_list[id_list.index('providers') + 1].lower() != 'microsoft.iotcentral':
+            raise InvalidArgumentValueError(
+                "Type must be Microsoft.IoTCentral/iotApps")
+        resource_group_name = id_list[id_list.index('resourceGroups') + 1]
+        app_name = id_list[id_list.index('iotApps') + 1]
+    else:
+        raise RequiredArgumentMissingError(
+            "Must provide id or application name, resource group, and type.")
+    return client.private_links.list(resource_group_name, app_name)
+
+
+def show_private_endpoint_connection(client, resource_group_name=None, connection_id=None, account_name=None, private_endpoint_connection_name=None):
+
+    return get_private_endpoint_connection(client=client,
+                                           resource_group_name=resource_group_name,
+                                           connection_id=connection_id,
+                                           account_name=account_name,
+                                           private_endpoint_connection_name=private_endpoint_connection_name,
+                                           return_args=False)
+
+
+def get_private_endpoint_connection(client, resource_group_name=None, connection_id=None, account_name=None, private_endpoint_connection_name=None, return_args=False):
+
+    if resource_group_name and account_name and private_endpoint_connection_name:
+        output = client.private_endpoint_connections.get(resource_group_name=resource_group_name,
+                                                         resource_name=account_name,
```

Comment:
NIT: We should keep naming consistent at all levels. Going from account_name to resource_name causes confusion.

Reply:
After looking more into the PR this makes sense now. account_name is for the parent resource (IoT App) while resource_name is used for the private endpoint connection resource.